seacrowd/sea_datasets/id_msvd/id_msvd.py

```diff
@@ -0,0 +1,138 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import (SCHEMA_TO_FEATURES, TASK_TO_SCHEMA,
+                                      Licenses, Tasks)
+
+_CITATION = """\
+@article{hendria2023msvd,
+    author    = {Willy Fitra Hendria},
+    title     = {MSVD-Indonesian: A Benchmark for Multimodal Video-Text Tasks in Indonesian},
+    journal   = {arXiv preprint arXiv:2306.11341},
+    year      = {2023},
+    url       = {https://arxiv.org/abs/2306.11341},
+}
+"""
+
+_DATASETNAME = "id_msvd"
+_DESCRIPTION = """\
+MSVD-Indonesian is derived from the MSVD (Microsoft Video Description) dataset, which is
+obtained with the help of a machine translation service (Google Translate API). This
+dataset can be used for multimodal video-text tasks, including text-to-video retrieval,
+video-to-text retrieval, and video captioning. Same as the original English dataset, the
+MSVD-Indonesian dataset contains about 80k video-text pairs.
+"""
+
+_HOMEPAGE = "https://github.com/willyfh/msvd-indonesian"
+_LANGUAGES = ["ind"]
+_LICENSE = Licenses.MIT.value
+_URLS = {
+    "text": "https://raw.githubusercontent.com/willyfh/msvd-indonesian/main/data/MSVD-indonesian.txt",
+    "video": "https://www.cs.utexas.edu/users/ml/clamp/videoDescription/YouTubeClips.tar" 
+}
+
+_SUPPORTED_TASKS = [Tasks.VIDEO_TO_TEXT_RETRIEVAL]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class IdMsvdDataset(datasets.GeneratorBasedBuilder):
+    """MSVD dataset with Indonesian translation."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    SEACROWD_SCHEMA_NAME = TASK_TO_SCHEMA[_SUPPORTED_TASKS[0]].lower()  # "vidtext"
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=f"{_DATASETNAME}",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "video_path": datasets.Value("string"),
+                    "texts": datasets.Sequence(datasets.Value("string")),
```

Comment:
in source, the data is a pair of video_path and single text
```suggestion
                    "text": datasets.Value("string"),
```